.circleci/cimodel/data/windows_build_definitions.py

```diff
@@ -0,0 +1,139 @@
+import cimodel.lib.miniutils as miniutils
+
+
+NON_PR_BRANCH_LIST = [
+    "master",
+    r"/ci-all\/.*/",
+    r"/release\/.*/",
+]
+
+
+class WindowJob:
+    def __init__(self,
+                 test_index,
+                 vscode_spec,
+                 cuda_version,
+                 force_on_cpu=False,
+                 run_on_prs_pred=lambda job: job.vscode_spec.year != 2019):
+
+        self.test_index = test_index
+        self.vscode_spec = vscode_spec
+        self.cuda_version = cuda_version
+        self.force_on_cpu = force_on_cpu
+        self.run_on_prs_pred = run_on_prs_pred
```

Comment:
It wouldn't compile otherwise, would it?
```suggestion
        self.run_on_prs_pred = run_on_prs_pred(self)
```

Reply:
Oh, sorry, I've realized you are using it as predicate (rather than boolean later)
IMO both are fine (I would prefer to resolve predicate to boolean in constructor, but lazy resolution is fine as well)